cfbs.json

```diff
@@ -737,6 +737,35 @@
         "json cfbs/def.json def.json"
       ]
     },
+    "stig-canonical-ubuntu-20-lts": {
+      "description": "Canonical Ubuntu 20.04 LTS Security Technical Implementation Guide",
+      "tags": ["security", "compliance", "ubuntu"],
+      "repo": "https://github.com/nickanderson/cfengine-security-hardening",
+      "by": "https://github.com/nickanderson",
+      "version": "0.0.1",
+      "commit": "bdc7a16c6e52ff44149eb2abfd510fcee12dd2c1",
+      "subdirectory": "./stig/canonical_ubuntu_20.04_lts/",
+      "dependencies": [
+        "compliance-report-imports",
+        "library-parsed-local-groups"
+      ],
+      "steps": [
+        "copy ./config.cf services/security-hardening/stig/ubuntu-20/config.cf",
```

Comment:
PHEW! Reviewing a module with separate policy files in separate dirs was quite time consuming. :O Hmm. Not sure how or if to improve that but the contained policy looks good to me.

Reply:
Curious, how closely did you review the policy against the stig?

Reply:
FWIW, I didn't expect you to review the policy against the stig, just to make sure no policy is devious.

Reply:
I didn't review against the stig at all. Only that it wasn't devious.